apps/site/src/views/Resources/sections/Landing/Landing.tsx

```diff
@@ -1,9 +1,12 @@
+import Image from "next/image";
+
 import styles from "./Landing.module.scss";
+import resourcesTitle from "@/assets/images/resources-title.svg";
```

Comment:
Chore: let's try to keep a consistent import order: library components, our custom components, assets, then stylesheets. See HeadingCard.tsx or StarterPacks.tsx for an example.